packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -136,4 +136,8 @@ const DevApp: FunctionComponent = () => {
   );
 };
 
-render(<DevApp />, node);
+if (node) {
+  createRoot(node).render(<DevApp />);
+} else {
+  throw new Error("Unable to find DOM element with id 'app'");
+}
```

Comment:
Leftover React 18 migration

Reply:
Is this necessary for this PR or you just doing it while you're here? Have we verified there are no issues?

Reply:
Just doing it while I'm here. There are no issues – this is not packaged with MBD, it's just the thing to test it out when developing it.